.travis.yml

```diff
@@ -37,20 +37,20 @@ addons:
     - yarn
 
 rvm:
-  - 2.3.4
```

Comment:
I guess you forgot to revert this.

Reply:
https://github.com/tootsuite/mastodon/pull/6032#discussion_r157369178

Reply:
In my opinion, the configuration should match with `Gemfile`. That is the reason I added the line in the commit which changed the version requirement in `Gemfile`.
I also wonder if the review https://github.com/tootsuite/mastodon/pull/6097#discussion_r158653842 meant to keep the CI configuration or not, so cc: @Gargron 

(For your information, I relaxed the version requirement for RHEL SCL users and CentOS SCL users. Now they have Ruby 2.4 and I use it. I guess the other latest operating systems which are able to run Mastodon have Ruby 2.4.)

Reply:
I'm OK with no longer testing against 2.3 but also keeping it compatible with the Gemfile. I feel like the 2.3 tests are redundant and just make CI take longer, but at the same time, if someone is still on 2.3, I don't want to *force* them to upgrade sooner than they would otherwise. It's very unlikely that we will start using Ruby features or syntax that is not already available in 2.3

Reply:
@Gargron Thanks! (I use Ruby 2.3.3)

Reply:
If you use Ruby 2.3.x should be more careful because nobody tests it on the version.
You may have to tests and fix some if it is broken even if you don't have to rewrite `Gemfile`.

Reply:
If you use Ruby 2.3.x should be more careful because nobody tests it on the version.
You may have to tests and fix some if it is broken even if you don't have to rewrite `Gemfile`.